server/src/test/java/org/opensearch/index/remote/RemoteSegmentTransferTrackerTests.java

```diff
@@ -403,99 +359,79 @@ public void testComputeBytesLag() {
     }
 
     public void testIsUploadBytesAverageReady() {
-        int uploadBytesMovingAverageWindowSize = pressureSettings.getUploadBytesMovingAverageWindowSize();
-        pressureTracker = new RemoteSegmentTransferTracker(
-            shardId,
-            directoryFileTransferTracker,
-            uploadBytesMovingAverageWindowSize,
-            pressureSettings.getUploadBytesPerSecMovingAverageWindowSize(),
-            pressureSettings.getUploadTimeMovingAverageWindowSize()
-        );
+        int movingAverageWindowSize = pressureSettings.getMovingAverageWindowSize();
+        pressureTracker = new RemoteSegmentTransferTracker(shardId, directoryFileTransferTracker, movingAverageWindowSize);
```

Comment:
nit - transferTracker (remove pressure word)

Reply:
Updated. Resolving this thread. Please feel free to re-open if needed.